src/components/app.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import Svg from './svg'
+import Nav from './nav'
+import SelectYear from './selectYear'
+import HistoryData from './historyData'
+import CurrentData from './currentData'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  'default': {
+    app: {
+      	height: '100%',
+      	backgroundColor: '#07263b',
+      	textAlign: 'center',
+      	color: 'white',
+      	paddingBottom: '0',
+    },
+    title: {
+      	paddingTop: '25px',
```

Comment:
No need to write 'px'. React will add that automatically. e.g. the following is preferred:

`passingTop: 25,`